Source/Foundation/bsfUtility/Utility/BsOctree.h

```diff
@@ -210,10 +203,11 @@ namespace bs
 			NodeElements mElements;
 
 			Node* mParent;
-			Node* mChildren[8];
+			Node* mChildren[8] = {  nullptr, nullptr, nullptr, nullptr,
+									nullptr, nullptr, nullptr, nullptr };
 
-			UINT32 mTotalNumElements : 31;
-			UINT32 mIsLeaf : 1;
+			UINT32 mTotalNumElements : 31 = 0;
```

Comment:
This bit didn't compile on newest MSVC, not sure what the standard says about that.

Reply:
Hrmm, weird. I was able to use this on gcc no problem.

Perhaps it's a gcc extension? 

I'll keep that in mind for later changes.